base/strings/io.jl

```diff
@@ -608,6 +608,46 @@ function indentation(str::AbstractString; tabwidth=8)
     count, true
 end
 
+"""
+    indent(str::AbstractString, n::Int)
+
+Indent each non-blank line by `n` spaces.
+
+# Examples
+```jldoctest
+julia> Base.indent("a\\nb", 4)
+"    a\\n    b"
+
+julia> Base.indent("  a\\n  \\n  b", 2)
+"    a\\n  \\n    b"
+```
+"""
+function indent(str::AbstractString, n::Int)
+    indent == 0 && return str
+    # Note: this loses the type of the original string
+    buf = IOBuffer(sizehint=sizeof(str))
+    indent_str = " " ^ n
```

Comment:
```suggestion
    indent_str = ' ' ^ n
```
Only a few ns faster, but a bit nicer.

Reply:
Hmmm, I thought you had to use a string, thanks.